tests/pytests/test_vecsim_svs.py

```diff
@@ -577,7 +580,19 @@ def test_gc():
 
         # Phase 2: Force garbage collection to reclaim memory
         # Explicit GC should reduce memory usage after marked deletions
+        orig_workers_stats = getWorkersThpoolStats(env)
         forceInvokeGC(env, DEFAULT_INDEX_NAME)
+        cur_workers_stats = getWorkersThpoolStats(env)
+
+        # GC background jobs for SVS should be either pending in low priority queue, in progress or already done.
+        env.assertEqual(orig_workers_stats['totalJobsDone'] + num_workers,
```

Comment:
Why not to pause the threadpool and get an exact number of `pending` (before pause) and `done` (after resume and wait for `numJobsInProgress)` ?